_studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp

```diff
@@ -786,7 +786,11 @@ mfxStatus VideoDECODEH265::GetDecodeStat(mfxDecodeStat *stat)
     m_stat.NumSkippedFrame = m_pH265VideoDecoder->GetSkipInfo().numberOfSkippedFrames;
     m_stat.NumCachedFrame = 0;
 
-    H265DecoderFrame *pFrame = m_pH265VideoDecoder->GetDPBList()->head();
+    H265DBPList *dpb = m_pH265VideoDecoder->GetDPBList();
+    if (!dpb)
+        return MFX_ERR_NULL_PTR;
```

Comment:
`MFX_ERR_NULL_PTR` is described as 
> NULL pointer in the input or output arguments
Having null pointer for DPB here looks to be unexpected, so I think `MFX_ERR_UNDEFINED_BEHAVIOR` status would be better.

Reply:
Fixed